magda-web-client/src/Pages/OtherPages.js

```diff
@@ -35,6 +35,10 @@ const renderBody = () => {
             <Route exact path="/projects" component={ProjectsViewer} />
             <Route exact path="/projects/new" component={CreateProject} />
             <Route path="/projects/:projectId" component={ProjectDetails} />
+            {/* hide in prod */}
```

Comment:
Feature flag? this flag could also be use to switch the mailto links to http links to this path if it is enabled